german/Schlafen/schlafen.md

```diff
@@ -0,0 +1,6 @@
+Glückwunsch! 
+
+Dazu …
+
+
+… benutzen Sie den [solarbetriebenen Motor](solarbetriebener-Motor/solarbetriebener-Motor.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.